api/openshift_info.py

```diff
@@ -0,0 +1,97 @@
+import os
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+from helper import helper
+
+
+class ServiceClass(object):
+    def __init__(self):
```

Comment:
Please add comments for every function. It should have 3 lines.
1. Purpose 
2. Inputs if any
3. Outputs

Reply:
I commented the code with meaningful comments
Every Comment Contains 
Purpose: <Purpose>
Input: <Input parameters>
Output: <Output>